### DISCUSSION TITLE
Tooltip trigger area

### DISCUSSION THREAD

**Discussion by akhchan99** [bootstrap-vue/bootstrap-vue#5819] [Q&A] [answered]
For tooltip (eg. default), you move your mouse over to the button, and the tooltip show up above the button.  But when you move your mouse above to the tooltip, the tooltip is still active.  Are there any flag to make the tooltip go hidden as soon as you move your mouse away from the button?  As the button is what trigger the tooltip, once its out of the button area, the tooltip should go away.  But currently, if you move the mouse to the tooltip placement, it remain shown.

**Comment by Hiws** [comment, marked as answer]
Use the `noninteractive` modifier or property.
<https://bootstrap-vue.org/docs/components/tooltip#noninteractive-tooltips>

#### Modifier
```html
<b-btn v-b-tooltip.noninteractive="'Hello World'">Tooltip</b-btn>
```

#### Prop
```html
<b-btn id="my-button">Tooltip</b-btn>
<b-tooltip target="my-button" noninteractive>
  Hello World
</b-tooltip>
```

**Comment by akhchan99** [comment]
Can it be set globally?

so that

`<b-btn v-b-tooltip="'Hello World'">Tooltip</b-btn>`

by default is non-interactive?

**Comment by Hiws** [comment]
Not that i know of.
